docs/guides/audio-config.md

```diff
@@ -54,8 +53,19 @@ An example to get PipeWire working on Ubuntu, which uses PulseAudio by default i
 The "Audio Volume" dialog / Audio in System Settings allow users to "Raise maximum volume", allowing to go past 100%. This
 does not offer a great deal of flexibility, it might work for getting acceptable recordings however.
 
+## Speakers
```

Comment:
I think it would be better to move this to a sub-heading under `## Speakers`. Something like `### MacBookPro16,1`.

Reply:
Can you close this